This sucks - but you try manually changing this in openoffice.lst
every time you want to do a release.

Index: scp/source/office/config.scp
===================================================================
RCS file: /cvs/installation/scp/source/office/Attic/config.scp,v
retrieving revision 1.19.40.1
diff -u -p -u -r1.19.40.1 config.scp
--- scp/source/office/config.scp	9 Jan 2004 18:36:50 -0000	1.19.40.1
+++ scp/source/office/config.scp	28 Apr 2004 10:40:48 -0000
@@ -8,5 +8,5 @@
 Installation GID_INSTALLATION
     ProductName = "${PRODUCTNAME}";
     ProductVersion = "${PRODUCTVERSION}";
-    ProductExtension = "${PRODUCTEXTENSION}";
+    ProductExtension = PRODUCTEXTENSION;
     VendorName = "OpenOffice.org";
Index: scp/source/office/makefile.mk
===================================================================
RCS file: /cvs/installation/scp/source/office/Attic/makefile.mk,v
retrieving revision 1.12.24.2
diff -u -p -u -r1.12.24.2 makefile.mk
--- scp/source/office/makefile.mk	25 Jan 2004 13:06:38 -0000	1.12.24.2
+++ scp/source/office/makefile.mk	28 Apr 2004 10:40:48 -0000
@@ -72,6 +72,12 @@ TARGETTYPE=CUI
 SCPDEFS+=-DENABLE_CRASHDUMP
 .ENDIF
 
+.IF "$(OOO_BUILD_EXTENSION)"!=""
+SCPDEFS+=-DPRODUCTEXTENSION=\""$(OOO_BUILD_EXTENSION)"\"
+.ELSE
+SCPDEFS+=-DPRODUCTEXTENSION=\"""\"
+.ENDIF
+
 # --- Settings -----------------------------------------------------
 
 .INCLUDE :  svpre.mk
